TicketingSystem/src/cassa/Ticket.java

```diff
@@ -0,0 +1,85 @@
+package cassa;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ticket {
+    protected TicketType type;
+    protected Status status;
+    protected int pieces;
+    protected static List<Ticket> seatTicketList;
+    protected static List<Ticket> vipTicketList;
+    protected static List<Ticket> standTicketList;
+
+    public Ticket() {
+        seatTicketList = new ArrayList<>();
+        vipTicketList = new ArrayList<>();
+        standTicketList = new ArrayList<>();
+    }
+
+    public Ticket(TicketType ticketType, Status status) {
+        this.type = ticketType;
+        this.status = status;
+        if (ticketType.equals(TicketType.SEAT)) {
+            seatTicketList.add(this);
+        } else if (ticketType.equals(TicketType.VIP)) {
+            vipTicketList.add(this);
+        } else if (ticketType.equals(TicketType.STANDING_TICKET)) {
+            standTicketList.add(this);
```

Comment:
We don't necessarily need different lists for different kind of tickets - is there any particular reason you have chosen to do it this way?